GAME/TYPES.H

```diff
@@ -143,7 +143,7 @@ enum weather_type
 };
 
 #if PC_VERSION
-#pragma pack(push, 1)
+
```

Comment:
Conditional C pre-processor code now has no purpose and should be removed.

Reply:
Done